popup.js

```diff
@@ -0,0 +1,86 @@
+/* eslint-disable no-plusplus */
```

Comment:
- [ ] Please do not disable linters. Linters help us write industry standard & uniform code in a team 👍🏾